src/Bicep.Core/Navigation/IArtifactReferenceSyntax.cs

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
+using Bicep.Core.Configuration;
+using Bicep.Core.Diagnostics;
```

Comment:
nit: Unused usings.

Reply:
No harm in calling these out, but it's also fine to ignore them, since the weekly `dotnet format` run should clean them up in an eventually consistent manner.